root/static/scripts/tests/typeInfo.js

```diff
@@ -4785,7 +4785,7 @@ MB.relationshipEditor.exportTypeInfo({
          "cardinality1" : 0,
          "deprecated" : false,
          "id" : 281,
-         "orderableDirection" : 0,
+         "orderableDirection" : 1,
```

Comment:
It’s which end of the relationship can be ordered. To understand what that means, assume we have these “is parent of” relationships:
- Mother is parent of A
- Mother is parent of B
- Father is parent of A
- Father is parent of B

We could now choose whether we want the ordering apply to the parents or to the children. In one case, we would see “children: 1. A, 2. B” on the pages for Mother and Father, and “parents: Father, Mother [no explicit order]” on the pages for A and B; in the other case, we would have “children: A, B [no explicit order]” on the pages for Mother and Father and “parents: 1. Mother, 2. Father” on the pages for A and B.

A value of 0 means that neither of the endpoints is ordered.